ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartOptionsEditor.tsx

```diff
@@ -32,6 +32,7 @@ import ChevronDown from 'mdi-material-ui/ChevronDown';
 import ChevronUp from 'mdi-material-ui/ChevronUp';
 import { TimeSeriesQueryDefinition } from '@perses-dev/core';
 import { OptionsEditorProps, TimeSeriesQueryEditor, usePlugin } from '@perses-dev/plugin-system';
+import { OptionsEditorTabs } from '../../components/OptionsEditorTabs';
```

Comment:
I created a `components` directory in `panels-plugin` because this seems very specific to panels and doesn't need to live in the generic `components` directory. Let me know if you disagree and would prefer it live there.